cadc-util/src/main/java/ca/nrc/cadc/net/AuthChallenge.java

```diff
@@ -0,0 +1,145 @@
+/*
+************************************************************************
+*******************  CANADIAN ASTRONOMY DATA CENTRE  *******************
+**************  CENTRE CANADIEN DE DONNÉES ASTRONOMIQUES  **************
+*
+*  (c) 2020.                            (c) 2020.
```

Comment:
(c) 2022

Reply:
doh, templates out of date because of working from home since 2020 :-)